elixir/apps/web/lib/web/live_hooks/fetch_subject.ex

```diff
@@ -26,7 +26,4 @@ defmodule Web.LiveHooks.FetchSubject do
   def on_mount(:default, _params, _session, socket) do
     {:cont, socket}
   end
-
-  defp context_type(%{"as" => "client"}), do: :client
```

Comment:
Router has been updated to not hit this plug/hook for clients since they don't need a subject during sign in.